test/CMakeLists.txt

```diff
@@ -83,6 +80,11 @@ if(WITH_GZFILEOP)
     list(APPEND TEST_SRCS test_gzio.cc)
 endif()
 
+find_package(Threads)
```

Comment:
In #1299, I use `target_sources` instead of trying to add to `TEST_SRCS`. It might be worth simplifying the two blocks of changes in this file into one like I have done.

Reply:
Done.